src/main/java/ch/heigvd/Main.java

```diff
@@ -1,18 +1,57 @@
 package ch.heigvd;
 
-public class Main {
-    // -i input
-    // -o output
-    // -r recursive
-    // -h help
-    // (empty , same as help)
-
-    public static void main(String[] args) {
-        // Recursive folder creation
-        Converter.convert("./image/webp.webp", "./image/a/b/output.png");
-        // Conversion png->webp
-        Converter.convert("./image/png.png", "./image/output.webp");
-        // Recursive conversion
-        Converter.convertRecursively("./image", "png", "webp");
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@Command(
+        name = "image file converter",
+        description = "convert jpg-png-webp format, with lossless if needed",
+        version = "1.0.0"
+)
+public class Main implements Callable<Integer> {
```

Comment:
This line throws me "Cannot resolve symbol 'Callable'" I fixed it locally by adding ```import java.util.concurrent.Callable;```